### DISCUSSION TITLE
Select/define user document folder in settings

### DISCUSSION THREAD

**Discussion by Manu-387-10** [OpenBoard-org/OpenBoard#1186] [Ideas]
Hello community.
Subject: Openboard V1.7.x file management
Context: I use openboard in combination with an interactive video projector in different rooms in my high school. Currently .ubz files are saved in the local directory of the room computer. So I no longer see them when I change rooms, it's problematic.
Greetings: would it be possible to add an option in the personalization settings allowing the possibility of adding a network path for saving documents? So you could keep your document library anywhere on the network.
Thank you for your possible consideration

**Comment by Manu-387-10** [comment]
My network security does not allow me to modify the 'OpenBoardUser.config' configuration file located in AppData (Windows)

**Reply by letsfindaway** [reply]
See also #1173 proposing to be able to define the data directory in a user writable location. I think this is closely related to the discussion here.
